src/OverlayTrigger.js

```diff
@@ -33,13 +33,15 @@ const OverlayTrigger = React.createClass({
     delayShow: React.PropTypes.number,
     delayHide: React.PropTypes.number,
     defaultOverlayShown: React.PropTypes.bool,
-    overlay: React.PropTypes.node.isRequired
+    overlay: React.PropTypes.node.isRequired,
+    containerPadding: React.PropTypes.number
```

Comment:
With this added prop can you include some documentation for it so other's know how/when to use it.